### ISSUE TITLE
display long filename properly

### ISSUE THREAD

**Issue by kswang1029** [CARTAvis/carta-frontend#1948]
user feedback from v3 beta test

Images from the ALMA Science Archive usually have long file names such as 

> member.uid___A001_X879_X8dc.J1924-2914_pol.spw0_1_2_3.mfs.IQU.manual.image.pbcor.fits

with the v3 release, mostly when the space is not sufficient, only the first few characters are displayed so sometimes we see identical file name from different images.

The request is to have a better way to handle long file names so that users can easily read the full file names to know which image is which. 

We may consider to use tooltip to show the full file name. For example:

https://user-images.githubusercontent.com/20819712/182782205-4a104f4d-194a-4a6d-8027-a19d7ae7d9d7.mov

We may also consider to show first part and last part of the long file name with ... in between like what macOS does.

**Comment by I-Chenn** [comment]
@kswang1029  @YuHsuan-Hwang:
I would like to discuss with you a bit first before making the first commit to this issue.

According to the description above, I am not sure which part of CARTA should be applied to the new function. Since the "**File browser**" is already with tooltip to show the file name (see below):
![image](https://user-images.githubusercontent.com/106953609/193741459-db9cff70-2cd5-418a-8a03-f0d309c99414.png)

Hence, I assume that the new function should be applied to the "**Image List**" layer, right? Please let me know if it's not.
I have made a quick modification about it, which is shown below:
![image](https://user-images.githubusercontent.com/106953609/193740483-8ceb31d1-ce85-4998-9105-a6fe02983953.png)
Do you think if the format is OK to go? Or should I use the same format of the "**File browser**"?

To summarize, I would like to ask:
1. Is this issue for "**Image List layer**"?
2. Is the tooltip format OK for you?

**Comment by kswang1029** [comment]
@I-Chenn we need the fix for the image list widget and the cursor info widget.
<img width="892" alt="Screen Shot 2022-10-04 at 14 03 25" src="https://user-images.githubusercontent.com/20819712/193746138-8fefbeff-351e-46b0-b605-349fb52d7fb1.png">

and can the format be like the following as we have for the file browser dialog?
<img width="677" alt="Screen Shot 2022-10-04 at 13 59 35" src="https://user-images.githubusercontent.com/20819712/193746207-980ab5fa-33d4-4871-a261-a0a47b1f70ac.png">

**Comment by I-Chenn** [comment]
OK, understood!
Thanks!
